sdk/iot/iot-model-parser/src/generated/aggregate_context.g.ts

```diff
@@ -0,0 +1,1332 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
this file seems to have been edited by a human. It should not because it is inside the `generated` directory. Expect all files under the generated directory to possibly be overwritten by the subsequent generations by autorest.

Reply:
Also, what is up with the `g` in the module file name?

Reply:
@deyaaeldeen this code has been generated but by the generation code in a separate repo. 

This brings up a q of: should the generation code be hosted in the Azure-SDK-for-JS folder, or not? I'm going with no for now, and generating the code from https://github.com/Azure/azure-iot-parser-node

We can update the generation code to generate with camelCase. The .g is to denote that these files are generated.

Reply:
@YoDaMa 
> should the generation code be hosted in the Azure-SDK-for-JS folder, or not? I'm going with no for now, and generating the code from https://github.com/Azure/azure-iot-parser-node

I do not have access to that repo and I am not sure what do you mean by generation code.

> We can update the generation code to generate with camelCase. The .g is to denote that these files are generated.

IIRC, the code gen always use camel case so I am not sure how did you customize it? using scripts? are these scripts the generation code you were referring to earlier?